docs/setup-cluster/slurm/slurm-known-issues.rst

```diff
@@ -381,6 +381,10 @@ Some constraints are due to differences in behavior between Docker and Singulari
  PBS Known Issues
 ******************
 
+-  If the ``Cluster`` tab on the DeterminedAI WebUI does not display the GPU information, there
+   could be an issue in the PBS configuration. Please refer to :ref:`PBS Requirements
```

Comment:
Can we provide a little more information as to what the specific issue may be?

Reply:
Please let me know what kind of information you would like to be added I can add it.

Reply:
I think this update to the documentation is in response to PBS GPU issue you just worked on where if not all vnodes have the same GPUs, then it shows "<various>", or something along those lines, right?  If so, where in the documentation you are adding is the solution to that problem described?

Reply:
Updated the ref link to point to the exact section for solving the issue.